pom.xml

```diff
@@ -138,4 +124,17 @@
             <!--</plugin>-->
         </plugins>
     </build>
+    <profiles>
```

Comment:
made this change to simplify local development in case when the re is no needs to release the project